python-backend/app/api/mines/namespace/mines.py

```diff
@@ -1,27 +1,27 @@
 from flask_restplus import Namespace
 
 from app.api.mines.mine.resources.mine_map import MineMapResource
-from ..mine.resources.mine import MineResource, MineListSearch, MineListResource
-from ..mine.resources.mine_type import MineTypeResource, MineTypeListResource
-from ..mine.resources.mine_type_detail import MineTypeDetailResource
-from ..mine.resources.mine_tenure_type_code import MineTenureTypeCodeResource
-from ..mine.resources.mine_disturbance_code import MineDisturbanceCodeResource
-from ..mine.resources.mine_commodity_code import MineCommodityCodeResource
-from ..status.resources.status import MineStatusResource, MineStatusListResource
-from ..region.resources.region import MineRegionResource
-from ..tailings.resources.tailings import MineTailingsStorageFacilityListResource
-from ..documents.mines.resources.mine_document_resource import MineDocumentListResource
-from ..compliance.resources.compliance import MineComplianceSummaryResource
-from ..compliance.resources.compliance_article import ComplianceArticleResource
-from ..mine.resources.mine_basicinfo import MineBasicInfoResource
+from app.api.mines.mine.resources.mine import MineResource, MineListSearch, MineListResource
```

Comment:
Are we now following the pattern of importing from the root as much a s possible.

Reply:
I just checked the change history. More recent changes were using the absolute path. So, I updated them all to be the same=